Components/Model.cs

```diff
@@ -40,6 +42,7 @@ public class Model : IModel
 
             _fileContentCache = new FileContentCache(fileContentCacheProvider);
             _cdtsEnvironments = new CdtsEnvironmentCache(cdtsCacheProvider).GetContent();
```

Comment:
Not that big of a deal in this case but might as well: The same `CdtsEnvironmentCache` object should be re-used for both `GetContent()` and `GetSRUHashes()`